src/main/java/teamrtg/rtg/client/GuiCustomizeWorldScreenRTG.java

```diff
@@ -0,0 +1,815 @@
+        this.list = new GuiPageButtonList(this.mc, this.width, this.height, 32, this.height - 32, 25, this, aguipagebuttonlist$guilistentry, aguipagebuttonlist$guilistentry1, aguipagebuttonlist$guilistentry2, aguipagebuttonlist$guilistentry3);
```

Comment:
These lines are excessively long and should be chopped down to an acceptable width limit. (In the old days that would mean a max of 72 columns, but that's a bit extreme for modern terminals.)